src/sentry/south_migrations/0365_backfill_first_project_heroku.py

```diff
@@ -0,0 +1,1001 @@
+# -*- coding: utf-8 -*-
+from sentry.models.organizationonboardingtask import OnboardingTask, OnboardingTaskStatus
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+from south.db import db
+from south.v2 import DataMigration
+
+
+class Migration(DataMigration):
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        feature_id = 60  # From featureadoption.py (there's no constant defined).
+        FeatureAdoption = orm['sentry.FeatureAdoption']
+        OrganizationOnboardingTask = orm['sentry.OrganizationOnboardingTask']
+        Organization = orm['sentry.Organization']
+        Project = orm['sentry.Project']
+
+        queryset = Organization.objects.all()
+        for org in RangeQuerySetWrapperWithProgressBar(queryset):
+            if Project.objects.filter(organization=org).exists():
+                FeatureAdoption.objects.create(
```

Comment:
Both of these need to be wrapped in a transaction block with a catch on IntegrityError.

Like:

```python
try:
  with transaction.atomic():
    FeatureAdoption.objects.create(...)
    OrganizationOnboardingTask.objects.create(...)
except IntegrityError:
  pass
```

This will allow the migration itself be idempotent and safe to race conditions.